gtwrap/pybind_wrapper.py

```diff
@@ -104,16 +104,10 @@ def _wrap_method(self,
                      '.def("deserialize", []({class_inst} self, string serialized)' \
                      '{{ gtsam::deserialize(serialized, *self); }}, py::arg("serialized"))' \
                        .format(class_inst=cpp_class + '*')
-            return serialize_method + deserialize_method
-
-        if cpp_method == "pickle":
-            if not cpp_class in self._serializing_classes:
-                raise ValueError(
-                    "Cannot pickle a class which is not serializable")
             pickle_method = self.method_indent + \
                 ".def(py::pickle({indent}    [](const {cpp_class} &a){{ /* __getstate__: Returns a string that encodes the state of the object */ return py::make_tuple(gtsam::serialize(a)); }},{indent}    [](py::tuple t){{ /* __setstate__ */ {cpp_class} obj; gtsam::deserialize(t[0].cast<std::string>(), obj); return obj; }}))"
-            return pickle_method.format(cpp_class=cpp_class,
-                                        indent=self.method_indent)
+            return serialize_method + deserialize_method + \
```

Comment:
I would (a) document _wrap_method, and (b) put the entire if block in another method, also documented. Very scarce documentation in these functions, in general... Makes it hard to parse if you have not yourself written the code

Reply:
Yeah docstrings for all the methods has been on my TODO list but not on my A->B :slightly_frowning_face: 
Perhaps another GSOC project.